### PR TITLE
[IMP] base,web,mail: new on-premise subscription flow

### PR DESCRIPTION
**Pull request by adr-odoo**

### CODE REVIEW

**Comment by dbo-odoo** [comment]
@julienlegros Review please, oh great guru?

File: addons/web/views/database_manager.html [hunk 1]
```diff
@@ -62,7 +59,20 @@
...
+        <div class="col-md-6">
+          <label for="country" class="control-label">Country</label>
+          <select name="country_id" class="form-control" required="required" autocomplete="off">
+              <option value="False">Country...</option>
+            {% for country in countries %}
+              <option value="{{ country[0] }}">{{ country[1] }}</option>
+            {% endfor %}
+            </select>
+        </div>
+      </div>
```
Comment (julienlegros) [inline addons/web/views/database_manager.html]: too much indent here (2 vs 4 space indentations).

File: addons/web_settings_dashboard/controllers/main.py [hunk 2]
```diff
@@ -34,6 +36,11 @@ def web_settings_dashboard_data(self, **kw):
             LIMIT 10
         """)
         pending_users = cr.fetchall()
+
+        # See update.py for this computation
+        limit_date = datetime.now() - timedelta(15)
+        enterprise_users = request.env['res.users'].search_count([("login_date", ">=", limit_date.strftime(DEFAULT_SERVER_DATETIME_FORMAT))])
```
Comment (julienlegros) [inline addons/web_settings_dashboard/controllers/main.py]: missing `["share", "=", false]` ?

Comment (dbo-odoo) [inline addons/web_settings_dashboard/controllers/main.py]: Yes indeed.

**Pull request closed without merging**